src/playGame.js

```diff
@@ -0,0 +1,25 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MESSAGES, RULES } from "../constants/messages.js";
+import randomNumber from "../utils/randomNumber.js";
+import userInputValidation from "../utils/userInputValidation.js";
+import resultPrint from "../utils/resultPrint.js";
+import compareToNumber from "../utils/compareToNumber.js";
+
+const playGame = async () => {
+  const comNum = randomNumber();
+  while (true) {
+    const userInput = await MissionUtils.Console.readLineAsync(
+      GAME_MESSAGES.INPUT
+    );
+    await userInputValidation(userInput);
+    const userNum = Array.from(userInput, Number);
```

Comment:
```suggestion
    const userNum = Array.from(userInput, Number);
```
저라면 `map` 함수 사용했을 것 같은데 `Array.from` 사용 엄청 좋은 것 같네요👍👏🙃

Reply:
저도 이번에 찾아보면서 처음 사용해 봤어요 ! Array.from 좋은 것 같습니다〰️